data/en/1_0005_updated.json

```diff
@@ -0,0 +1,2168 @@
+                                "when": [{
+                                    "id": "pay-pattern-frequency-confirmation-answer",
+                                    "condition": "equals",
+                                    "value": "No, I would like to add another pay pattern"
+                                }]
+                            }
+                        },
+                        {
+                            "goto": {
+                                "group": "weekly-pay",
+                                "when": [{
+                                    "id": "pay-pattern-frequency-answer",
+                                    "condition": "contains",
+                                    "value": "Weekly"
+                                }]
+                            }
+                        },
+                        {
+                            "goto": {
+                                "group": "fortnightly-pay",
+                                "when": [{
+                                    "id": "pay-pattern-frequency-answer",
+                                    "condition": "contains",
+                                    "value": "Fortnightly"
+                                }]
+                            }
+                        },
+                        {
+                            "goto": {
+                                "group": "calendar-monthly",
+                                "when": [{
+                                    "id": "pay-pattern-frequency-answer",
+                                    "condition": "contains",
+                                    "value": "Calendar monthly"
+                                }]
+                            }
+                        },
+                        {
+                            "goto": {
+                                "group": "four-weekly-pay",
+                                "when": [{
+                                    "id": "pay-pattern-frequency-answer",
+                                    "condition": "contains",
+                                    "value": "Four weekly"
+                                }]
+                            }
+                        },
+                        {
+                            "goto": {
+                                "group": "five-weekly-pay",
+                                "when": [{
+                                    "id": "pay-pattern-frequency-answer",
+                                    "condition": "contains",
+                                    "value": "Five weekly"
+                                }]
+                            }
+                        }
+                    ]
+                }
+            ],
+            "id": "pay-pattern",
+            "title": "Pay Pattern"
+        },
+        {
+            "blocks": [{
+                    "id": "weekly-pay-introduction",
+                    "title": "Weekly Pay",
```

Comment:
The spec has page and question numbers? but we don't have them any more?

Reply:
I think this was discussed before but we decided to remove section numbers

Reply:
@shahi645 therefore the specs should reflect this.